lightkurve/interact.py

```diff
@@ -24,7 +24,10 @@
 from astropy.stats import sigma_clip
 from astropy.utils.exceptions import AstropyUserWarning
 
-from .utils import KeplerQualityFlags, LightkurveWarning
+from .utils import KeplerQualityFlags, TessQualityFlags, LightkurveWarning
+import os
+from astropy.coordinates import SkyCoord, Angle
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'SkyCoord' from line 23](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4911672)